doc/source/schedule-backup.rst

```diff
@@ -0,0 +1,47 @@
+.. _schedule: 
+
+Schedule backups
+==============================
+
+In |PBM| version 1.4.1 and earlier, you can use ``crond`` or similar services to schedule backup snapshots.
+
+.. important::
+
+   Before configuring ``crond``, make sure that you have :ref:`installed <install>` and :ref:`configured <initial-setup>` |PBM| to make backups in your database. Start a backup manually to verify this: :command:`pbm backup`. 
+
+The recommended approach is to create a :file:`crontab` file in the :dir:`/ect/cron.d` directory and specify the command in it. This simplifies server administration especially if multiple users have access to it.
+
+As an example, let's configure to run backup snapshots on 23:30 every Sunday. 
+
+The steps are the following:
+
+1. Create a ``crontab`` file. Let's name it ``pbm-backup``. 
+2. Specify the command in the file:
+   
+   .. code-block:: guess
+   
+      30 23 * * sun root /usr/bin/pbm backup --mongodb-uri="mongodb://pbmuser:secretpwd@localhost1:27018,localhost2:27018,localhost3:27018/?replSetName=xxx"
+
+3. Verify that backups are running in :file:`/var/log/cron` or :file:`/var/log/syslog` logs:
+   
+   .. code-block:: guess
+   
+      $ grep CRON /var/log/syslog
+
+.. note:: 
+
+   Previous backups are not automatically removed from the backup storage. 
+   Therefore, timely remove them to clean up the storage. 
+
+Schedule backups with |PITR|
+---------------------------------------------------
+
+ You can both enable incremental backups for :ref:`pitr` and run backup snapshots on a schedule using ``crond``. The steps are the following:
+
+ 1. Enable point-in-time recovery
+ 2. Make a fresh backup manually. It will serve as the starting point for incremental backups
+ 3. Configure ``crond`` to run backup snapshots on a schedule
```

Comment:
```suggestion
  Once you are running full backup snapshots on a schedule using ``crond`` it becomes suitable to enable :ref:`pitr`. The steps below can be done in any order, but PITR will only start running after at least one full backup has been made.

 * Enable point-in-time recovery
 * Make a fresh backup manually. It will serve as the starting point for incremental backups
 * Configure ``crond`` to run backup snapshots on a schedule
```